dashboard/src/components/DataLoader.js

```diff
@@ -147,6 +147,7 @@ export default function DataLoader() {
 
             if (stats.persons) newList.push('person');
             if (stats.groups) newList.push('group');
+            if (stats.consultations) newList.push('consultation');
             if (['admin', 'normal'].includes(user.role)) {
               if (stats.consultations) newList.push('consultation');
```

Comment:
Alors là ya un problème on push deux fois consultations. Si on veut bien l'ajouter cc @arnaudambro je pense qu'il faut le retirer d'en dessous.

```suggestion
            if (stats.consultations) newList.push('consultation');
            if (['admin', 'normal'].includes(user.role)) {
```

Reply:
@marioonb oui je suis d'accord, tu peux "commit suggestion" :)

Reply:
ah mince... bien vu !! j'etais persuadé d'avoir supprimé